src/components/ClubHome/Daily/DailyControl.js

```diff
@@ -16,81 +16,102 @@ const DailyControl = ({
   setColor,
   getInfo,
 }) => {
-  const onClickModify = () => {
+  let token;
+
+  const onClickModify = (el) => {
     setTitle(el.title);
     setPeriod([el.startDate, el.endDate]);
     setNo(el.no);
     setColor(el.colorCode);
     setPop(3);
```

Comment:
```suggestion
    setPop(3);
```
여기에 3이 들어간 이유는 뭘까용? 이런 값들은 알아보기 쉽게 변수로 할당해주는게 좋을거 같네요